how-to-guides/data-versioning/notebooks/Version_datasets_in_model_training_runs.ipynb

```diff
@@ -168,7 +168,9 @@
    "source": [
     "import neptune.new as neptune\n",
     "\n",
-    "run = neptune.init(project=\"common/data-versioning\", api_token=\"ANONYMOUS\")"
+    "run = neptune.init(\n",
```

Comment:
I guess `init()` will be updated separately?

Reply:
yup